ace-internal.d.ts

```diff
@@ -1292,6 +1287,13 @@ export namespace Ace {
     export interface ScrollbarEvents {
         "scroll": (e: { data: number }) => void;
     }
+
+    export interface AriaOptions {
```

Comment:
I think this interface is fairly specific to text input, do you think it could make sense to name it accordingly? (e.g. something like `TextInputAriaOptions`)